### ISSUE TITLE
Not getting start and end index for answer

### ISSUE THREAD

**Issue by lalitr994** [TheAtticusProject/cuad#19]
Right now I am able to get start logits and end logits from the model output. but these logits contain features and examples.
How can I get the start and end index for answers so, that I can mark those on the context in the form of a bounding box.

@TheAtticusProject
